linera-chain/src/chain.rs

```diff
@@ -713,22 +753,6 @@ where
                 ChainError::ClosedChain
             );
         }
-
-        // The first incoming message of any child chain must be `OpenChain`. A root chain must
-        // already be initialized
-        if block.height == BlockHeight::ZERO
-            && self
-                .execution_state
-                .system
-                .description
-                .get()
-                .map_or(true, |description| description.is_child())
-        {
-            ensure!(
-                block.starts_with_open_chain(),
```

Comment:
Is this helper method still used after this change?

Reply:
Good catch, I'll remove it!